espnet/espnet.c

```diff
@@ -508,9 +549,7 @@ wifi_osi_funcs_t g_wifi_osi_funcs = {
 	._get_random = _get_random,
 	._get_time = _get_time,
 	._random = _random,
-// #if CONFIG_IDF_TARGET_ESP32S2 || CONFIG_IDF_TARGET_ESP32S3 || CONFIG_IDF_TARGET_ESP32C3
-//     uint32_t (* _slowclk_cal_get
-// #endif
+	._slowclk_cal_get = _slowclk_cal_get,
```

Comment:
I think it's better to use the commented code:

```c

#if CONFIG_IDF_TARGET_ESP32S2 || CONFIG_IDF_TARGET_ESP32S3 || CONFIG_IDF_TARGET_ESP32C3
    ._slowclk_cal_get = _slowclk_cal_get,
#endif
```

and define `CONFIG_IDF_TARGET_ESP32C3` in the CFLAGS (`-DCONFIG_IDF_TARGET_ESP32C3`).